core/notifications/notifications.go

```diff
@@ -0,0 +1,126 @@
+package notifications
+
+import (
+	"fmt"
+
+	"github.com/anyproto/any-sync/app"
+
+	"github.com/anyproto/anytype-heart/core/event"
+	"github.com/anyproto/anytype-heart/pb"
+	"github.com/anyproto/anytype-heart/pkg/lib/pb/model"
+)
+
+const CName = "notificationService"
+
+type Notifications interface {
+	app.Component
+	CreateAndSendLocal(notification *model.Notification) error
+	UpdateAndSend(notification *model.Notification) error
```

Comment:
could client code update notification id and timestamp?

Reply:
- Well, I planned to use it in middleware in case when notification objects is updated. And in this case we didn't update these fields in notification. We can add logic to keep id and timestamp, but now this function is supposed to be used inside middleware in only one case